src/components/GaugeChart/handleSeries.js

```
@@ -547,11 +572,12 @@ function setSeriesInit(seriesUnit, iChartOption) {
  * @param {数据} data
  * @returns
  */
-function handleSeries(iChartOption,optionColor,containerWidth) {
+export function handleSeries(iChartOption,optionColor,containerWidth,containerHeight) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)